services/fluxmonitor.go

```diff
@@ -190,7 +192,7 @@ func (fm *FluxMonitor) stopTickers() {
 
 func (fm *FluxMonitor) eventListener(ch <-chan interface{}) {
 	defer fm.Stop()
-
+	logger.Info("FM listening for events")
```

Comment:
Should use the FM logger so we get more context in the log lines.

```suggestion
	fm.logger.Info("FM listening for events")
```